.github/ISSUE_TEMPLATE/other.yml

```
@@ -20,3 +20,8 @@ body:
       description: Is there anything else we should know about this issue?
     validations:
       required: false
+  - type: markdown
+    attributes:
+      value: |
+        You can also join the Discord community [here](http://discord.eddiehub.org)
```

'our' word would look better 🤔 .
Other LGTM 🚀 
```suggestion
        You can also join our Discord community [here](http://discord.eddiehub.org)
```

---

I like that, thanks for the suggestion. This might need to be updated in other issue forms, too. 🙂